uiServices/src/services_test.go

```diff
@@ -120,6 +120,9 @@ func TestValidJsonPost(t *testing.T) {
 	reader := strings.NewReader(validJson)
 	r.HandleFunc("/configs", postConfigs)
 
+	//remove file if exists
+	os.Remove(os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/ServiceTestConfig.xml")
```

Comment:
maybe consider mocking out the file creation and use the godog tests to test actual creation of the file. not essential for this PR but worth investigating.